packages/react/CHANGELOG.md

```diff
@@ -43,14 +43,14 @@ All notable changes to this project will be documented in this file. See
 
 # 1.0.0-rc.2 (2020-08-09)
 
-**Note:** Version bump only for package @chakra-ui/core
+**Note:** Version bump only for package @chakra-ui/react
 
 # Change Log
 
 All notable changes to this project will be documented in this file. See
 [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [1.0.0-rc.1](https://github.com/chakra-ui/chakra-ui/compare/@chakra-ui/core@1.0.0-rc.0...@chakra-ui/core@1.0.0-rc.1) (2020-08-06)
+# [1.0.0-rc.1](https://github.com/chakra-ui/chakra-ui/compare/@chakra-ui/react@1.0.0-rc.0...@chakra-ui/react@1.0.0-rc.1) (2020-08-06)
```

Comment:
These `compare` links will fail since they're tag-based and those tags don't exist.

Reply:
Good call. Removed the changes from this file